curriculum/challenges/english/01-responsive-web-design/learn-css-grid-by-building-a-magazine/step-005.md

```diff
@@ -56,7 +56,7 @@ assert(document.querySelector('p')?.className === 'hero-subtitle');
 Your `p` element should have the text set to `Our efforts to restructure our curriculum with a more project-based focus`.
 
 ```js
-assert(document.querySelector('p')?.textContent === 'Our efforts to restructure our curriculum with a more project-based focus');
+assert.equal(document.querySelector('p')?.textContent?.trim(), 'Our efforts to restructure our curriculum with a more project-based focus');
```

Comment:
```suggestion
assert.equal(document.querySelector('p')?.textContent?.trim().replace(/\s{2,}/, ' '), 'Our efforts to restructure our curriculum with a more project-based focus');
```

Reply:
I think too lenient is better than too strict, in this case - we're not teaching them how to write, we're teaching CSS here.